scripts/check_base_rowcounts.py

```
@@ -0,0 +1,91 @@
+    except (urllib.error.URLError, ValueError, TimeoutError) as e:
+        print(f"::warning::row-count manifest unreachable ({e}) -- skipping base check")
+        return None
+
+
+def local_counts(src: str, tables):
+    conn = sqlite3.connect(src)
+    out = {}
+    try:
+        for t in tables:
+            try:
+                out[t] = conn.execute(f'SELECT COUNT(*) FROM "{t}"').fetchone()[0]
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)